### ISSUE TITLE
Docker Firefox node says my extension is not correctly signed.

### ISSUE THREAD

**Issue by prasanth-G24** [SeleniumHQ/docker-selenium#970]
## 🐛 Bug Report

I am not able to load extensions while using firefox docker selenium image. If I use the crx version of the same extension with chrome docker selenium image or use the same xpi in [the selenium hub](https://bit.ly/2TlkRyu), everything works fine.

## Expected behavior

I am using an extension to set data in the localstorage and retrieve it using python. As the extension is not getting loaded, I am getting None value in python. (I am getting expected value in chrome and non docker selenium hub as mentioned earlier).

## Obtained error

> 1570538439757	addons.xpi-utils	WARN	Add-on performance@site24x7.com is not correctly signed.
> 1570538439757	addons.xpi-utils	WARN	addMetadata: Add-on performance@site24x7.com is invalid: Error: Extension performance@site24x7.com is not correctly signed(resource://gre/modules/addons/XPIDatabase.jsm:2452:17) JS Stack trace: addMetadata@XPIDatabase.jsm:2452:1

TLDR; The docker logs say my extension is not correctly signed.

## manifest.json

```
{
  "name": "URL writer",
  "version": "1.0",
  "description": "Writes url to localstorage ",
  "applications": {
    "gecko": {
      "id": "performance@site24x7.com"
    }
  },
  "author" : "prasanth",
  "manifest_version": 2,
  "permissions":[
  	"tabs",
  	"activeTab",
  	"webNavigation",
  	"storage",
  	"webRequest",
  	"webRequestBlocking",
  	"<all_urls>"
  ],
    "background": {
    	"scripts": ["background.js"]
    },
    "content_scripts":[
    	{
    		"matches": ["<all_urls>"],
    		"js": ["app.js"],
    		"run_at": "document_idle"
    	}
    ]
}
```

## Environment

OS: Ubuntu 18.04
Docker-Selenium image version:  3.141.59-iron
Firefox Image ID: 60ebe2031f12
Docker version: 18.09.7
Docker-Compose version: 2
Exact Docker command to start the containers:

```
version: '2'
services:
  firefox:
    image: selenium/node-firefox
    depends_on:
      - hub
    environment:
      HUB_HOST: hub
      NODE_MAX_INSTANCES: 5
      NODE_MAX_SESSION: 5
    volumes:
      - /dev/shm:/dev/shm
  chrome:
    image: selenium/node-chrome
    depends_on:
      - hub
    environment:
      HUB_HOST: hub
      NODE_MAX_INSTANCES: 5
      NODE_MAX_SESSION: 5
    volumes:
      - /dev/shm:/dev/shm 

  hub:
    image: selenium/hub
    ports:
      - "4444:4444"
```

**Comment by prasanth-G24** [comment]
> [mozilla/geckodriver#473 (comment)](https://github.com/mozilla/geckodriver/issues/473#issuecomment-283298262)

Thank you, closing this thread.
